packages/core/src/Tag/stories/SelectableControlled.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from "react";
+import { HvTag, HvTypography, theme } from "@hitachivantara/uikit-react-core";
+
+import { css } from "@emotion/css";
+
+export const SelectableControlled = () => {
+  const [selectedTags, setSelectedTags] = useState<any>(["Asset 1", "Asset 3"]);
+
+  const handleSelect = (
+    event: React.MouseEvent<HTMLElement>,
+    tag?: React.ReactNode,
+    selected?: boolean
+  ) => {
+    if (selected) {
+      setSelectedTags((prev) => [...prev, tag]);
+    } else {
+      setSelectedTags((prev) => prev.filter((item: any) => item !== tag));
+    }
+  };
+
+  return (
+    <div
+      className={css({
+        display: "flex",
+        flexDirection: "column",
+        gap: theme.space.sm,
+      })}
+    >
+      <div className={css({ display: "flex", gap: theme.space.sm })}>
+        <HvTag
+          label="Asset 1"
+          color="cat1"
+          selectable
+          selected={selectedTags.includes("Asset 1")}
+          onDelete={() => {
+            alert("On Delete Action");
+          }}
+          onClick={handleSelect}
+        />
```

Comment:
We could create an array of `tags` with `label`/`color?`/`bgcolor`, and `map` over them to reduce duplication - (likely) similarly to what apps will be doing

We also have a `TagArray` sample that... doesn't use an array? Shouldn't it? 🤔 


```suggestion
      {tags.map(({label, color, bgColor}) => (
        <HvTag
          label={label}
          color={bgColor}
          selectable
          selected={selectedTags.includes(label)}
          className={css({ color: color ? theme.colors[color] : undefined })}
          onDelete={() => alert("On Delete Action")}
          onClick={(evt, selected) => handleSelect(evt, selected, label)}
        />
```